### PR TITLE
Removing unused imports, cells and __future__ from the notebooks.

### PR DESCRIPTION
**Pull request by gabrieldemarmiesse**
V2 of #895

I didn't find any better way of being diff friendly than than to edit the json directly :(

### CODE REVIEW

File: docs/tutorials/losses_triplet.ipynb [hunk 1]
```diff
@@ -144,19 +144,6 @@
         "## Setup"
```
Comment (WindQAQ) [inline docs/tutorials/losses_triplet.ipynb]: <p>could you remove redundant line 2 and 3?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: docs/tutorials/networks_seq2seq_nmt.ipynb [hunk 2]
```diff
@@ -108,19 +108,6 @@
         "# **Setup**"
```
Comment (WindQAQ) [inline docs/tutorials/networks_seq2seq_nmt.ipynb]: <p>move line 9 to line 6 and remove trailing empty lines</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (WindQAQ) [inline docs/tutorials/networks_seq2seq_nmt.ipynb]: <pre class="ql-syntax" spellcheck="false">import string
import re
from unicodedata import normalize

from tensorflow.keras.preprocessing.text import Tokenizer
from tensorflow.keras.preprocessing.sequence import pad_sequences
from tensorflow.keras.utils import to_categorical
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import LSTM
from tensorflow.keras.layers import Dense
from tensorflow.keras.layers import Embedding
from tensorflow.keras.layers import RepeatVector
from tensorflow.keras.layers import TimeDistributed
from tensorflow.keras.callbacks import ModelCheckpoint

from numpy import array
from numpy import argmax
from nltk.translate.bleu_score import corpus_bleu
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (WindQAQ) [inline docs/tutorials/networks_seq2seq_nmt.ipynb]: <p>could you move the imports to <code>Additional Resources</code> section?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

File: docs/tutorials/optimizers_conditionalgradient.ipynb [hunk 3]
```diff
@@ -107,19 +107,6 @@
         "## Setup"
```
Comment (WindQAQ) [inline docs/tutorials/optimizers_conditionalgradient.ipynb]: <pre class="ql-syntax" spellcheck="false">import tensorflow_addons as tfa

from matplotlib import pyplot as plt
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

File: docs/tutorials/optimizers_lazyadam.ipynb [hunk 4]
```diff
@@ -103,19 +103,6 @@
     "## Setup"
```
Comment (WindQAQ) [inline docs/tutorials/optimizers_lazyadam.ipynb]: <pre class="ql-syntax" spellcheck="false">import tensorflow_addons as tfa
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

File: docs/tutorials/time_stopping.ipynb [hunk 5]
```diff
@@ -96,10 +96,8 @@
    "metadata": {},
```
Comment (WindQAQ) [inline docs/tutorials/time_stopping.ipynb]: <pre class="ql-syntax" spellcheck="false">import tensorflow_addons as tfa

from tensorflow.keras.datasets import mnist
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense, Dropout, Flatten
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

File: docs/tutorials/tqdm_progress_bar.ipynb [hunk 6]
```diff
@@ -157,7 +157,6 @@
         "\n",
```
Comment (WindQAQ) [inline docs/tutorials/tqdm_progress_bar.ipynb]: <pre class="ql-syntax" spellcheck="false">import tensorflow_addons as tfa

from tensorflow.keras.datasets import mnist
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense, Dropout, Flatten
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: docs/tutorials/networks_seq2seq_nmt.ipynb (hunk 2 above)
Comment (WindQAQ) [inline docs/tutorials/networks_seq2seq_nmt.ipynb]: <p>Could you move imports to <code>Additional Resources</code>  sections?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (gabrieldemarmiesse) [reply docs/tutorials/networks_seq2seq_nmt.ipynb]: Can you expand on this? It seems that the imports are already in the `Additional Resources` section.

Comment (WindQAQ) [reply docs/tutorials/networks_seq2seq_nmt.ipynb]: Sure, I mean move
```python
from pickle import load
from numpy.random import shuffle
```
to this cell
```python
import string
import re
...
```

Comment (WindQAQ) [reply docs/tutorials/networks_seq2seq_nmt.ipynb]: Sure, I mean move
```python
from pickle import load
from numpy.random import shuffle
```
to this cell
```python
import string
import re
...
```
